lib/raven/client.rb

```diff
@@ -25,7 +25,7 @@ def send_event(event)
       event = event.to_hash
 
       unless @state.should_try?
-        failed_send(nil, event)
+        configuration.logger.error "Not sending event due to previous failure(s)."
```

Comment:
It would be cool if we could log here how long we have until the next attempt.

Reply:
It's also important that the backtrace (essentially all the info on line 96) is still printed here.

For people that don't use i.e. Sidekiq to send their Sentry events (so, automated retry), they're going to lose this event forever.

Reply:
Ok will look into it.